ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -105,6 +105,16 @@
   public Gauge<Long> requestResponseHandlerThreadRunning;
   public Gauge<Integer> activeOperations;
 
+  // metrics for tracking blob sizes and chunking.
+  public final Histogram putBlobSizeBytes;
+  public final Histogram putBlobChunkCount;
+  public final Histogram getBlobSizeBytes;
+  public final Histogram getBlobChunkCount;
+  public final Counter simpleBlobPutCount;
```

Comment:
Is the intention was counter or rate ? Can we start using Meters to track rates instead of counters?